python/dask_cudf/dask_cudf/core.py

```diff
@@ -414,6 +415,10 @@ def groupby(self, *args, **kwargs):
 
         return CudfSeriesGroupBy(self, *args, **kwargs)
 
+    @property
+    def list(self):
+        return ListMethods(self)
```

Comment:
cc @isVoid

If I remember correctly we needed to make this an attribute or cache it in some way since this property would instantiate a new `ListMethods` instance each time you access the property

Reply:
Creating a `ListMethods` instance should be free-ish though?

Reply:
But the `_column` underneath a `Series` isn't immutable, so if we cache `ListMethods` instances, we have to make sure we invalidate them when the columns are mutated. I'm really not sure we can do this easily today.